data/articles.csv

```diff
@@ -11,16 +11,16 @@ WordGrinder,WordGrinder: Distraction-Free Writing From the Command Line,http://f
+OnionShare,OnionShare - A Tool To Share Any Size Of Files Securely And Anonymously,https://www.2daygeek.com/onionshare-secure-way-to-share-files-sharing-tool-linux/,2017-12-04,"When we wrote an article about file sharing utilities such as Magic Womhole & transfer.sh most of the users asked is it secure? and where the files are stored. So, we decided to bring one of the best utility that stores and share files securely over the Internet. When we dig on that, we came to know about OnionShare which fulfill all this requirements."
 goobook,GooBook - Access Your Google Contacts From Commandline In Linux,https://www.ostechnix.com/goobook-access-google-contacts-commandline-linux/,2017-07-10,Here is an useful and handy utility called GooBook that allows you to access your Google contacts from commandline. The main purpose of Goobook is to access google contacts from commandline or a command line Email client like Mutt.
 archwiki,How To Search The Arch Wiki Website Right From Terminal,http://www.2daygeek.com/search-arch-wiki-website-command-line-terminal/#,2017-08-09,"Whenever you search anything about Arch Linux in web browser, I damn sure the first link would be Arch Wiki. Am i right? Yes, since Arch wiki is an ideal place to get solution for most of the arch Linux issues ..."
 googler,Googler A Command Line Tool To Search Google,http://www.linuxandubuntu.com/home/googler-a-command-line-tool-to-search-google,2017-08-09,"Looking for information on the web has become something that we do day in and day out. Whether it is support you are looking for, looking for a product or news about a particular topic, all you have to do is search on the web."
-lolcat,lolcat : tool to rainbow color linux terminal,http://kerneltalks.com/linux/lolcat-tool-to-rainbow-color-linux-terminal/,2017-07-10,Paint your command outputs with rainbow color!  Use lolcat (Ruby gem) tool and add some spice to black putty terminal!
+lolcat,lolcat : tool to rainbow color linux terminal,http://kerneltalks.com/linux/lolcat-tool-to-rainbow-color-linux-terminal/,2017-07-10,Paint your command outputs with rainbow color! Use lolcat (Ruby gem) tool and add some spice to black putty terminal!
 neofetch,Display Linux Distribution Logo in ASCII Art in Terminal,https://itsfoss.com/display-linux-logo-in-ascii/,2017-10-07,... You might have seen people sharing such desktop screenshot with terminal showing the logo of the Linux distribution used in ASCII art.
 pdd,pdd - Tool to find Date and Time Difference in Linux Command Line,https://linoxide.com/linux-how-to/pdd-tool-date-time-difference-linux-command-line/,2017-08-09,"In some occasions where you want to check by how many years someone older than you, how old you are (in days, years or months), the countdown to an event or the next flash sale. There is a python-based command line application ..."
```

Comment:
```suggestion
 pdd,pdd - Tool to find Date and Time Difference in Linux Command Line,https://linoxide.com/linux-how-to/pdd-tool-date-time-difference-linux-command-line/,2017-08-09,"In some occasions where you want to check by how many years someone older than you, how old you are (in days, years or months), the countdown to an event or the next flash sale. There is a Python-based command line application ..."
```
Maybe there are other cases of "python"?

Reply:
I'll check, thanks